docker/Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG CUDA="9.0"
 ARG CUDNN="7"
+ARG FORCE_CUDA="1"
```

Comment:
this will go out of context as soon as the `FROM` is called, so it needs to be further down the line.